react-prebuild.jsx

```diff
@@ -0,0 +1,25 @@
+// @flow
+import React from 'react';
+import {renderToString} from 'react-dom/server';
+import Logo from './kuma/javascript/src/Logo';
+
+let stdinAccumulator = '';
+
+process.stdin.on('data', input => {
+    stdinAccumulator += input.toString();
+});
+
+process.stdin.on('end', () => {
+    const locales = JSON.parse(stdinAccumulator);
+
+    const result = Object.keys(locales).reduce((acc, locale) => ({
+        [locale]: {
+            Logo: renderToString(<Logo
+                url={locales[locale].Logo.props.url}
+            />),
+        },
+        ...acc
```

Comment:
Object interpolation in a loop is a stealth way to get O(n^2) performance, so I avoid this pattern. (I also don't like using reduce() because I find it hard to understand.) But those are my style preferences. I can see that it would be easy to rewrite this loop to avoid those things.